sdk/containerregistry/test-resources.json

```diff
@@ -1,63 +1,70 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "registryName": {
+    "baseName": {
+      "type": "string",
       "defaultValue": "[resourceGroup().name]",
-      "type": "string"
+      "metadata": {
+        "description": "The base resource name."
+      }
+    },
+    "tenantId": {
+      "type": "string",
+      "metadata": {
+        "description": "The tenant ID to which the application and resources belong."
+      }
     },
-    "skuName": {
-      "defaultValue": "Basic",
-      "type": "string"
+    "testApplicationOid": {
+      "type": "string",
+      "metadata": {
+        "description": "The client OID to grant access to test resources."
+      }
     },
-    "skuTier": {
-      "defaultValue": "Basic",
-      "type": "string"
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "The location of the resource. By default, this is the same as the resource group."
+      }
     }
   },
   "variables": {
-    "apiVersion": "2020-11-01-preview",
-    "endpoint": "[concat('https://', parameters('baseName'), '.azurecr.io')]"
+    "apiVersion": "2019-05-01",
+    "endpointValue": "[format('https://{0}.azurecr.io', parameters('baseName'))]"
   },
   "resources": [
     {
       "type": "Microsoft.ContainerRegistry/registries",
```

Comment:
>registries [](start = 43, length = 10)

This template only create the base registry.  Do we not want to include other things here?

Reply:
Same answer as above. I haven't tested this yet.